docs/source/api_reference/models.rst

```diff
@@ -641,6 +641,11 @@ MultitaskIRVClassifier
 .. autoclass:: deepchem.models.torch_models.MultitaskIRVClassifier
   :members:
 
+HNN
+----------------
+.. autoclass:: deepchem.models.torch_models.HNN
```

Comment:
This is the wrong place to add this import since it is a nn.module and not a TorchModel.